TESTING.md

```diff
@@ -11,72 +11,82 @@ Recipeezy
 
 # Automated Test Cases:
 
+PRECONDITION TO RUN ALL TESTS:
+1. create a new environment
+2. run `pip install -r requirements.txt` in environment to get requirements
 ---
 Use case name
     Verify login with valid user name and password
 Description
-    Test the Google login page
+    Test for logging in only. This tests for access to the User database and
+    return to the HOME page.
 Pre-conditions
-    User has valid user name and password
+    No preconditions
 Test steps
-    1. Navigate to login page
-    2. Provide valid user name
-    3. Provide valid password
-    4. Click login button
+    1. run app using `python recipeezy.py`
+    2. open browser and delete cookies
+    3. go to the url given by app
+    4. click on CREATE
+    5. In Username, type "New_User"
+    6. In Password, type "Password"
+    7. Click submit button
 Expected result
-    User should be able to login
+    User should login and be returned to HOME page
 Actual result
-    User is notified of succesful login and is allowed to return to home
+    User is logged in and returned to HOME page
 Status (Pass/Fail)
     Pass
 Notes
-    N/A
+    * User is not given notification of user being logged in. It will show up on flask CLI and as a flash() message for debugging.
+    * Cookies will save your credentials so if you click on CREATE and aren't redirected, clear cookies and return to homepage.
+    * New account creation is not tested here, we are basically using a user that is manually added to the db and checking for validation.
 Post-conditions
     User is validated with database and successfully signed into their account.
-    The account session details are logged in database.
 
 ---
 Use case name
     Add a recipe to the database
 Description
-    Test recipe submission by adding a test recipe to the database
+    Test recipe submission by adding a test recipe to the database and checking for re-submission
 Pre-conditions
     User has a valid user name and password and logged in
 Test steps
-    1. Navigate to login page
+    1. Navigate to LOGIN page
     2. Provide valid user name
     3. Provide valid password
-    4. Click login button
-    5. Navigate to create page
+    4. Click submit button
+    5. Navigate to CREATE page
```

Comment:
create form does not appear even after logging in

Reply:
did you type in the correct username and password? It's hard-coded b/c create account isn't implemented yet.